.github/pull_request_template.md

```diff
@@ -0,0 +1,16 @@
+| Question      | Answer
+| ------------- | ---
+| Tickets       | <!-- URLs to GitHub or JIRA issue(s) (or N/A) -->
```

Comment:
Well... I wouldn't advertise:
1) Possibility of making one PR for several tickets
2) Using Github for issues (unless we've changed approach, there were some discussions, I don't remember conclusions).

Of course could be adjusted when needed (e.g. bug fixes, someone prefered GitHub issue anyway, etc.), I just wouldn't put it there by default.

Reply:
> Possibility of making one PR for several tickets

Even if few different tickets are related to single issue (ie. wrong SiteAccess configuration or something?).

> Using Github for issues (unless we've changed approach, there were some discussions, I don't remember conclusions).

We have to disable `Issues` tab in GitHub repository configuration then. But I don't have permissions to do so, ping: @lserwatka 

Personally I do like GitHub issues over JIRA for external reports, it's closer to common workflow of Open Source community but if it's general policy I don't mind removing it.

Reply:
> > Possibility of making one PR for several tickets
> Even if few different tickets are related to single issue (ie. wrong SiteAccess configuration or something?).

In that case you can edit template to hint that. I'm saying that hinting to the contributor that this is default way is not a best practise. Usually when you create PR it is related to one thing. There are exceptions, but as mentioned - we can adjust wording in such case - not by default.

Reply:
If you create a PR you already have a solution. It's not encouraging or hinting anyone to anything. It's just saying that if it's related to one or more tickets (and that may happen), this is the place where to place links. And that's all.

Like here: https://github.com/ezsystems/ezpublish-kernel/pull/2153

We may granulate it to separate rows, but is it really necessary? If PR solves to many things at the same time it's matter of Code Review, and not PR description template. :P

Reply:
> Like here: ezsystems/ezpublish-kernel#2153
> We may granulate it to separate rows, but is it really necessary? If PR solves to many things at the same time it's matter of Code Review, and not PR description template. :P

In the case you've linked, yes, it is. PR solving an issue, and issues related to that PR are two different concepts and should be separated.